int32 nFrames
float64[] Values
bool isEmpty
bool isPartial
